### ISSUE TITLE
meson build files missing in release

### ISSUE THREAD

**Issue by Sylensky** [Irqbalance/irqbalance#316]
Is there any particiular reason why the meson build files are missing in the relase archive?

**Comment by nhorman** [comment]
yes, because the meson build file was moved to the root directory after the most recent release was published.

**Comment by Sylensky** [comment]
Ack. are these files in the root directory for upcoming releases?

**Comment by nhorman** [comment]
They were moved in commit 922ee47bbfb44a1ee7346f434092028adf4dffcd , and will be available in the root directory in the next release.  Though theres no reason you can't use them from the contrib directory in the current release:
```
cd meson
meson setup <builddir> .
```
